src/core/components/info.jsx

```diff
@@ -100,17 +101,20 @@ export default class Info extends React.Component {
     basePath: PropTypes.string,
     externalDocs: ImPropTypes.map,
     getComponent: PropTypes.func.isRequired,
+    oas3selectors: PropTypes.func,
+    selectedServer: PropTypes.string,
   }
 
   render() {
-    let { info, url, host, basePath, getComponent, externalDocs } = this.props
+    let { info, url, host, basePath, getComponent, externalDocs, selectedServer } = this.props
     let version = info.get("version")
     let description = info.get("description")
     let title = info.get("title")
-    let termsOfService = info.get("termsOfService")
+    let termsOfService = buildUrl(info.get("termsOfService"), selectedServer)
```

Comment:
**Change request**: can this be `termsOfServiceUrl` instead, for consistency with `externalDocsUrl` below? I know it wasn't introduced here but it would be nice 😄